rules/windows/credential_access_dump_registry_hives.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to export a registry hive which may contain credentials using the Windows reg.exe tool.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+max_signals = 33
+name = "Credential Acquisition via Registry Hive Dumping"
+references = ["https://medium.com/threatpunter/detecting-attempts-to-steal-passwords-from-the-registry-7512674487f8"]
+risk_score = 73
+rule_id = "a7e7bfa3-088e-4f13-b29e-3986e0e756b8"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+ process.pe.original_file_name == "reg.exe" and
+ process.args in ("save", "export") and
+ process.args in ("hklm\\sam", "hklm\\security") and
+ not wildcard(process.parent.executable, "C:\\Program Files*\\Rapid7\\Insight Agent\\components\\insight_agent\\*\\ir_agent.exe")
```

Comment:
@Samirbous `in` is case sensitive, so you will need to use `:`.

```suggestion
 process.args : ("save", "export") and
 process.args : ("hklm\\sam", "hklm\\security") and
 not process.parent.executable : "C:\\Program Files*\\Rapid7\\Insight Agent\\components\\insight_agent\\*\\ir_agent.exe"
```

Reply:
already adjusted ;)

Reply:
error on 7.10, maybe will be fixed in 7.11 ?

![image](https://user-images.githubusercontent.com/64742097/100027198-9f1de380-2dec-11eb-8cb0-af138530e6ec.png)

Reply:
this works : a or :b instead on :(a, b) 

![image](https://user-images.githubusercontent.com/64742097/100027822-d9d44b80-2ded-11eb-96a5-b8f07b1261c8.png)

Reply:
correct, 7.10 doesn't have this, but `master` does

Reply:
this repo now accepts it as well, so if you make changes, it'll still pass tests